src/overlays/actors/ovl_Boss_Goma/z_boss_goma.c

```diff
@@ -1690,7 +1690,7 @@ void BossGoma_CeilingMoveToCenter(BossGoma* this, PlayState* play) {
 
         if (angle < this->actor.wallYaw) {
             absDiff = this->actor.wallYaw - angle;
-            angle = angle + absDiff / 2;
+            angle += absDiff / 2;
```

Comment:
I think x=x+ is better in this case for symmetry with the other branch
Note: am biased, wrote this 😔

Reply:
Yeah, I can see how the symmetry is nice here